Library/Formula/acpica.rb

```diff
@@ -2,8 +2,8 @@ class Acpica < Formula
   desc "OS-independent implementation of the ACPI specification"
   homepage "https://www.acpica.org/"
   head "https://github.com/acpica/acpica.git"
-  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150204.tar.gz"
-  sha1 "8c5514b1171afb40dca40289581c8ba3f17583e1"
+  url "https://acpica.org/sites/acpica/files/acpica-unix2-20150619.tar_1.gz"
+  sha1 "283c594c4b89db8e3bef8fca25021c52938987ad"
```

Comment:
Could you possibly make this `sha256 "12d68f781c9f5c3dfb61ef280f41f0d019d7ba4e914ef916ed3e1d22712ffd52"` instead? Thanks!